packages/base-map/src/index.tsx

```diff
@@ -155,16 +151,8 @@ const BaseMap = ({
         (!!baseLayer &&
           typeof baseLayer === "object" &&
           baseLayer.length > 1)) && (
-        <Styled.LayerSelector
-          className="filter-group"
-          id="filter-group"
-          onBlur={() => setFakeHover(false)}
-          onFocus={() => setFakeHover(true)}
-          onTouchEnd={() => setFakeHover(true)}
```

Comment:
I'm not longer able to open the layer selector when using a device without a mouse. Am I doing something wrong?

Reply:
That was a mistake. Reinstated in https://github.com/opentripplanner/otp-ui/pull/578/commits/8c32bb4c8f950822c2ff724d31e0efaf3d25db14.